### DISCUSSION TITLE
Welcome to pythonSoftIOC Discussions!

### DISCUSSION THREAD

**Discussion by Araneidae** [dls-controls/pythonSoftIOC#70] [Announcements]
## Welcome!

Having seen a number of issues which are not actually problems or feature requests in the library, I think it's time to open up the Discussions section!

I'll go through the conversational issues and move them over here.

**Comment by Robatronic** [comment]
Thanks!!

I was feeling a little bad about putting some of my questions under "Issues".   :)
